app/server/models/transfer.py

```diff
@@ -0,0 +1,611 @@
+import datetime, enum
```

Comment:
this file is still massive... couldn't split it out because of circular dependencies. how do we fix that? ideally each model has its own file so it's easier to think through it

Reply:
is this an okay name / grouping for what is the relation between these various models in the meantime?

Reply:
This is looking good. In my mind, the core models are:
- User
- TransferAccount
- CreditTransfer
- (Maybe) Organisation

Most other models are here to support those four.
So ideally I think we'd seperate out "TransferAccount" and "CreditTransfer" into two different files if dependencies permit, though i can imagine it's non trivial